native/cocos/editor-support/spine-wasm/spine-model.cpp

```diff
@@ -1,11 +1,14 @@
 #include "spine-model.h"
 
 SpineModel::SpineModel() {
+    SpineModel::data = new std::vector<uint32_t>(6, 0);
```

Comment:
`SpineModel` is unneeded.
And why not just define `std::vector<uint32_t> _data` in header file and call ` _data.resize(6, 0);` here?

Reply:
Defined as a pointer because it can be called directly at the binding code. Otherwise, an additional conversion is required at the time of the call. Example: :

Conversion required:
.function("getData", optional_override([](SpineModel &obj){
            return &obj.getData(); }), allow_raw_pointer<std::vector<unsigned int>>());

Use directly:
.function("getData", &SpineModel::getData, allow_raw_pointer<std::vector<unsigned int>>());

If you use references directly, you will get copy behavior and need to manually call delete() on the TS side.